Easydict/Feature/ViewController/View/ResultView/EZResultView.m

```diff
@@ -14,12 +14,14 @@
 #import "NSImage+EZResize.h"
 #import "NSImage+EZSymbolmage.h"
 #import "EZWindowManager.h"
+#import "EZOpenAILikeService.h"
 
 @interface EZResultView ()
 
 @property (nonatomic, strong) NSView *topBarView;
 @property (nonatomic, strong) NSImageView *serviceIcon;
 @property (nonatomic, strong) NSTextField *serviceNameLabel;
+@property (nonatomic, strong) EZButton *serviceModelButton;
```

Comment:
I saw that you replaced `NSTextField` with `EZButton`, is there any issue with using a label?

Reply:
No issues with a label, but I changed the appearance to have a background and cornerRadius, the button seems easy to customize.

Reply:
It's ok to use button here.

I just thought that we can use this button to switch models easily. When the button is clicked, an `NSMenu` will pop up, which is very useful for quickly switching models.

<img width="439" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/50bf73e8-eaee-4b5c-ab50-db5b27e545da">

Reply:
It will be easy to switch the model for openAI service since it has pre defined models. But for custom openai service, there is no predefined model list. I will check if have any good ways to handle this.